docs/docs/index.md

```diff
@@ -1,34 +1,35 @@
 
-Each module is built using the build technology that is either associated with the module type by default or is configured explicitly. Then the results of each module's build are packaged into an archive together with the multitarget application deployment descriptor. The deployment descriptor is created based on the structure of the multitarget application project, the multitarget application development descriptor (mta.yaml), the extension descriptors, if they are supplied (the builder merges them with mta.yaml), and the command line options. In the deployment descriptor, the design-time modules are converted into the module types that are recognized by the respective target platforms. The resulting multitarget application archive can then be deployed on the required target platform.
-See [Module types conversion](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/platform_cfg.yaml) for more details on how design time module types in `mta.yaml` are converted into deployment time module types in `mtad.yaml` according to the target platform. 
+Each module is built using the build technology that is either associated with the module type by default or is configured explicitly. Then the results of each module's build are packaged into an archive together with the multitarget application deployment descriptor. The deployment descriptor is created based on the structure of the multitarget application project, the multitarget application development descriptor (`mta.yaml` file), the extension descriptors, if they are supplied (the builder merges them with the `mta.yaml` file), and the command-line options. In the deployment descriptor, the design-time modules are converted into the module types that are recognized by the respective target platforms. The resulting multitarget application archive can then be deployed on the required target platform.
 
-If the MTA Build Tool encounters a module type in the `mta.yaml` that is not listed in the configuration file, its definition is copied to `mtad.yaml` as is and the `zip` builder is applied to the module (i.e. archived module sources will be packaged as the build result).
+For more details about how the design-time module types in the `mta.yaml` file are converted into deployment time module types in the `mtad.yaml` file according to the target platform, see [Module types conversion](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/platform_cfg.yaml). 
 
-All resource definitions are passed to the `mtad.yaml` file as is without mapping and validations (e.g., if the resource is supported in the target platform or specified parameters match the type).
+If the MTA Build Tool encounters a module type in the `mta.yaml` file that is not listed in the configuration file, its definition is copied to the `mtad.yaml` file as is and the `.zip` builder is applied to the module (in other words, the archived module sources are packaged as the build result).
```

Comment:
"zip" is the actual name of the builder. So it should not be replaced with ".zip"

Reply:
Ok, thanks for the info. Will you make the change or should I?

Reply:
Please change it. Thank you

Reply:
I cannot see a way to make that change now. I suggest that @ShimiT should merge the changes and then I will do an extra review.

Reply:
No problem. I will change it.

Reply:
Thanks